### PR TITLE
[Updates] plant portal

### PR DESCRIPTION
**Pull request by mtekman**
TODO:

* [X] Fixed video icon link
* [X] Added Understanding barcodes video tutorial
* [X] Added Plant youtube video tutorial
* [ ] Add miRNA links
* [x] Add Plant SVG logo
* [ ] Fix captions for plant

### CODE REVIEW

**Comment by mtekman** [comment]
related to https://github.com/usegalaxy-eu/website/pull/628

**Comment by mtekman** [comment]
Screenshot_2021-04-13 Galaxy Europe

Looks good I think

**Comment by bgruening** [comment]
The tutorials section can now be removed, isn't it?

**Comment by mtekman** [comment]
Screenshot_2021-04-13 Galaxy Europe

@bgruening  Does this look better, I merged into the above?

**Comment by bgruening** [comment]
Yes! Thanks.

File: index-plants.md [hunk 1]
```diff
@@ -26,17 +26,13 @@ The server is part if the European Galaxy server and is maintained by the [RNA B
 # Get started
 
 Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take [a guided tour]({{ page.website }}/tours/core.galaxy_ui){:target="_blank"} through Galaxy's user interface.
+Some relevant materials prior to the analysis of plant-related data is the analysis of [Chloroplast genome assembly](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/chloroplast-assembly/tutorial.html).
+
 
 # Training and Workshops
 
 We are passionate about training. So we are working in close collaboration with the [Galaxy Training Network (GTN)](https://galaxyproject.org/teach/gtn/){:target="_blank"} to develop training materials of data analyses based on Galaxy {% cite batut2017community %}. These materials hosted on the GTN GitHub repository are available online at [https://training.galaxyproject.org](https://training.galaxyproject.org){:target="_blank"}.
```
Comment (beatrizserrano) [inline index-plants.md:34]: what about adding this link? https://training.galaxyproject.org/training-material/search?query=plant

Comment (beatrizserrano) [reply index-plants.md:34]: @mtekman can you unify the labels plant and plants? should [this other tutorial](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.html) be labelled as plants too?

Comment (mtekman) [reply index-plants.md:33]: Added it, I moved this down to the main material section

Screenshot_2021-04-13 Galaxy Europe

**Pull request merged**